### PR TITLE
bugfix: double-redirection to home page to fix auth issue in optimize

### PR DESCRIPTION
**Pull request by buccarel**
## Description

Context:
https://github.com/camunda/camunda/issues/25644

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,7 +39,6 @@ faces-config.NavData
 .gradle
 .worktree
 work/
-public/
```
Comment (RomanJRW) [inline .gitignore:42]: This should not be removed?

**Review by RomanJRW — CHANGES_REQUESTED** [review]
Your commit message should just be `fix:...` rather than `bugfix:...`

File: .gitignore (hunk 1 above)
Comment (buccarel) [reply .gitignore:42]: I need too add the new html file in a directory that is called public. Maybe I can add the file bypassing the gitignore then?

Comment (RomanJRW) [reply .gitignore:42]: maybe it's ok if nothing else is getting committed by removing this line

**Review by RomanJRW — APPROVED** [review]

**Pull request merged**